fix: Fix attribute in Uninstalling Che (#2189)
codenvy/che-docs: modules/installation-guide/partials/proc_uninstalling-che-after-operatorhub-installation-using-openshift-cli.adoc

$ oc project _<{prod-id-short}_project>_
----

> . Obtain the {prod-short} cluster name. The following shows a cluster named `{prod-deployment}`:

Reviewer: ```suggestion
. Obtain the `checluster` Custom Resource name. The following shows a `checluster` Custom Resource named `{prod-checluster}`:
```

+
[subs="+quotes,attributes"]
----
$ oc get checluster
NAME          AGE

> {prod-deployment}   27m

Reviewer: ```suggestion
{prod-checluster}   27m
```

----

. Delete the {prod-short} cluster:
+
[subs="+quotes,attributes"]
----
$ oc delete checluster {prod-deployment}

> checluster.org.eclipse.che "{prod-deployment}" deleted

Reviewer: ```suggestion
$ oc delete checluster {prod-checluster}
checluster.org.eclipse.che "{prod-checluster}" deleted
```
